packages/next/src/export/routes/app-page.ts

```diff
@@ -114,7 +119,7 @@ export async function exportAppPage(
 
     // When PPR is enabled, we should grab the headers from the mocked response
     // and add it to the headers.
-    if (renderOpts.experimental.ppr) {
+    if (renderOpts.experimental.supportsPPR) {
```

Comment:
should this be `pprEnabled`? Having a hard time determining when something should be checking for `pprEnabled` or `supportsPPR`. Feels easy to mess up 

(If `supporstPPR` is correct here, we should update the comment above to not mention the word "enabled")